kbfsgit/runner.go

```diff
@@ -74,6 +75,10 @@ const (
 	// push, we actually fetch from the local repo and write the
 	// objects into the bare repo.
 	localRepoRemoteName = "local"
+
+	// The minimum number of refs needed to initialize a repo with a
+	// packed-refs file.
+	packedRefsThreshDefault = 10
```

Comment:
Is this measured or arbitrary?

Reply:
Arbitrary, but #1211 got rid of it anyway, so the point is moot.